config/prod/JKG-s3.yaml

```diff
@@ -0,0 +1,36 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template_path: remote/s3-bucket.yaml
+stack_name: GatesKI-TestStudy-Bucket
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "sysbio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad"
+  # The resource owner
+  OwnerEmail: "jake.gockley@sagebase.org"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username, required if AllowWriteBucket=true
+  SynapseUserName: 'jgockley'
+  # (Optional) Allow accounts, groups, and users to access bucket.
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+  # (Optional) true (default) to encrypt bucket, false for no encryption
```

Comment:
these are all optional meaning you do not need to specify them if you just want the defaults.  did you really want to change all of the optional parameters?